README.md

```diff
@@ -117,6 +118,7 @@ const advancePlayer = (move, player) =>
     Right: () => ({x: player.x + 1, y: player.y}),
     Down: () => ({x: player.x, y: player.y + 1}),
     Left: () => ({x: player.x - 1, y: player.y}),
+    _: () => (player),
```

Comment:
I like that it is consistent with the lines above, but not strongly opinionated about it.